pkg/vpcmodel/nodesExplainability.go

```diff
@@ -102,6 +104,21 @@ func (c *VPCConfig) getVsiNode(name string) Node {
 	return nil
 }
 
+// given a string address in non-cidr format return the list of all network interfaces attached to it
+func (c *VPCConfig) GetNetworkInterfaces(internalAddress string) (networkInterfaceNodes []Node, err error) {
```

Comment:
how is this function going to be used by explainability?
is it also part of this PR?

Reply:
Given an internal address, explainability will be applied to any of its VSIs (could be a Cartesian product if both src and dst given as internal cidr). If there are no VSIs connected then the user will  be provided with this explanation 
Lets finalize details in https://github.com/np-guard/vpc-network-config-analyzer/issues/353

No, it will be in a separate PR